js/templates/modals/listingDetail/shippingOptions.html

```diff
@@ -5,17 +5,16 @@
     <% } %>
     <div class="rowLg tx4">
       <div class="flexRow row">
-        <h3 class="col8"><%= option.name %></h3>
+        <h3 class="col8 rowLg"><%= option.name %></h3>
```

Comment:
The rowLg should go on the flexRow parent, but that's because of the changes I made after this. I'll fix it in my branch.